ajax/libs/videojs-vast-plugin/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "videojs-vast-plugin",
+  "version": "0.2.1",
+  "description": "A VAST plugin for VideoJS",
+  "filename": "videojs.vast.min.js",
+  "homepage": "https://theonion.github.io/videojs-vast-plugin/",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/theonion/videojs-vast-plugin.git"
+  },
+  "keywords": [
+    "video",
+    "videojs",
+    "vast"
+  ],
+  "license": "MIT",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/theonion/videojs-vast-plugin.git",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "videojs.vast.*"
+        ]
+      },
+      {
+        "basePath": "lib",
```

Comment:
The files under `lib` belongs other projects: [videojs-contrib-ads](https://github.com/videojs/videojs-contrib-ads) & [vast-client-js](https://github.com/dailymotion/vast-client-js).
The former one had been added, which can be found here https://cdnjs.com/libraries/videojs-contrib-ads.
The latter one should be added independently (Open a issue or directly open a pull request and cc this pull request). I will help open a issue for this in a minute this time.

So please remove L27 to L32. Thank you.